res/enkf/enkf_main.py

```diff
@@ -463,3 +462,10 @@ def addNode(self, enkf_config_node):
     def rng(self) -> RandomNumberGenerator:
         "Will return the random number generator used for updates."
         return self._get_shared_rng()
+
+
+from res._lib.local.enkf_main import get_local_config
+
+_RealEnKFMain._get_local_config = get_local_config
+
+del get_local_config
```

Comment:
Is it possible to inline this in `getLocalConfig`?

Reply:
OK